lib/nexpose/vulnerability.rb

```diff
@@ -124,7 +124,9 @@ def cleanup_html(source)
       result.gsub!(/<URLLink LinkTitle=\"(.*?)\" LinkURL=\"(.*?)\"\/>/i) { "\"#{$1.strip}\":#{$2.strip} " }
       result.gsub!(/<URLLink LinkURL=\"(.*?)\" LinkTitle=\"(.*?)\"\/>/i) { "\"#{$2.strip}\":#{$1.strip} " }
       result.gsub!(/<URLLink(.*)LinkURL=\"(.*?)\"(.*?)>(.*?)<\/URLLink>/m) {|m| "\"#{$4.strip}\":#{$2.strip} " }
-
+      result.gsub!(/&gt;/, '>')
```

Comment:
@rachkor I was thinking on not adding this and change the specs to check for `&gt;`/`&lt;`. I say so because it looks like in Dradis we are already displaying this ok.

Reply:
I had to add this because the `&gt;/&lt;` characters were displaying that way in Dradis (and breaking the specs in the process!)

Reply:
Can you add a spec for the HTML entities please?

Reply:
https://github.com/dradis/dradis-nexpose/pull/19/commits/d6e1a93abe84a9bb8d6a2b2f62e1d367c9c4a8eb